backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -128,4 +129,29 @@ class EakApplicationMutationService {
             }
         }
     }
+
+    @GraphQLDescription("Updates a note of an application")
+    fun updateApplicationNote(
+        applicationId: Int,
+        noteText: String,
+        dfe: DataFetchingEnvironment
+    ): Boolean {
+        val context = dfe.getContext<GraphQLContext>()
+        val jwtPayload = context.enforceSignedIn()
+
+        return transaction {
+            val application = ApplicationEntity.findById(applicationId) ?: throw UnauthorizedException()
+            // We throw an UnauthorizedException here, as we do not know whether there was an application with id
+            // `applicationId` and whether this application was contained in the user's project & region.
+
+            val user = AdministratorEntity.findById(jwtPayload.adminId)
+                ?: throw UnauthorizedException()
+
+            if (!mayViewApplicationsInRegion(user, application.regionId.value)) {
```

Comment:
we should probably use/add a function called `mayUpdateApplicationsInRegion` or so

Reply:
ok but the roles are the same for that so i thought its redundant to add a second function